API Blueprint Specification.md

```diff
@@ -801,32 +780,35 @@ Where:
 
 ```
 + Parameters
-    + id ... Id of a post.
+    + id - Id of a post.
 ```
 
 ```
 + Parameters
-    + id (number) ... Id of a post.
+    + id (number) - Id of a post.
 ```
 
 ```
 + Parameters
-    + id (required, number, `1001`) ... Id of a post.
+    + id: `1001` (number, required) - Id of a post.
 ```
 
 ```
 + Parameters
-    + id = `20` (optional, number, `1001`) ... Id of a post.
+    + id: `1001` (number) - Id of a post.
+        + Default: `20`
 ```
 
+You can give an enumeration of possible values using [MSON][] as shown below:
```

Comment:
Please do not refer to MSON here. Explain it without referencing to MSON syntax.